api/config/config_local.json

```diff
@@ -100,8 +100,7 @@
       "expiryDaysWarningThresholds": [1, 3, 7, 15, 30],
       "lookbackPeriod": 330,
       "trainingLookbackPeriod": 30
-    },
-    "tiersVisibleToUsers": ["registered", "controlled"]
```

Comment:
Unfortunately we can't remove this yet.  The reason is that the API+UI deploy non-atomically.

The API is updated before the UI, so there's a window where the old API is still referenced (by the old-version UI) before the UI can be updated to refer to the new API.

Therefore, our release/deployment safety process is:
* independent UI changes can be made at any time
* change/removal of API structures/endpoints which the UI depends on (like here) can only be done after (1) the UI no longer refers to it and (2) a release cycle has occurred since the UI change was made.

Please let me know if you have any questions.  There are probably some edge cases I'm missing here.